tensorboard/components/tf_paginated_view/tf-category-paginated-view.ts

```diff
@@ -28,6 +28,8 @@ import {
 import {TfDomRepeat} from './tf-dom-repeat';
 
 @customElement('tf-category-paginated-view')
+// @ts-ignore(go/ts48upgrade) Fix code and remove this comment. Error:
```

Comment:
Might be worth making this "(googlers see go/ts48upgrade)" here and in the other files as well.

Reply:
I prefer to keep as it was written internally to ensure we keep compatibility with whatever tool parses @ts-ignore. We'll be hopefully upgrading to TS 4.8 this week anyway so these won't last long.